src/Compilers/CSharp/Portable/Symbols/NativeIntegerTypeSymbol.cs

```diff
@@ -63,7 +63,7 @@ internal NativeIntegerTypeSymbol(NamedTypeSymbol underlying) : base(underlying,
 
         internal override NamedTypeSymbol GetDeclaredBaseType(ConsList<TypeSymbol> basesBeingResolved) => _underlyingType.GetDeclaredBaseType(basesBeingResolved);
 
-        internal override ImmutableArray<NamedTypeSymbol> GetDeclaredInterfaces(ConsList<TypeSymbol> basesBeingResolved) => _underlyingType.GetDeclaredInterfaces(basesBeingResolved);
+        internal override ImmutableArray<NamedTypeSymbol> GetDeclaredInterfaces(ConsList<TypeSymbol> basesBeingResolved) => MakeInterfaces(_underlyingType.GetDeclaredInterfaces(basesBeingResolved));
```

Comment:
>MakeInterfaces(_underlyingType.GetDeclaredInterfaces(basesBeingResolved)); [](start = 124, length = 74)

I think we would want to cache the result and not recalculate it over and over again. Also, I think we don't need to maintain two different sets between ```GetDeclaredInterfaces``` and ```InterfacesNoUseSiteDiagnostics``` because this type is never defined in source with its own list of interfaces. I think it would be fine to always return result based on ```_underlyingType.InterfacesNoUseSiteDiagnostics```.